android/app/src/main/java/com/facebook/f8/MainActivity.java

```diff
@@ -89,6 +94,11 @@ protected boolean getUseDeveloperSupport() {
 
       return Arrays.<ReactPackage>asList(
         new MainReactPackage(),
+        new RNSharePackage(),
+        new RNSendIntentPackage(),
+        new LinearGradientPackage(),
+        new FBSDKPackage(),
+        new CodePush(null, this, BuildConfig.DEBUG),
```

Comment:
@rozele I guess I must have. I must have missed that in all the noise from applying the gitignore.